docs/source/operations/howto/backend-deployment.rst

```diff
@@ -106,6 +106,8 @@ To configure your values:
              enabled: true
              cacert: orchestrator-cacert
 
+#. Optional: If the orchestrator requires your backend to be authenticated by using mTLS follow :doc:`mtls-setup`.
```

Comment:
Can we change the "How to" files order to have this reference pointing to a previous link ?

Reply:
So the order would be orchestrator > mTLS > backend ?

Reply:
Mmmh no sorry I'm must have been confused, the order seems fine this way...

Reply:
Maybe put the how by order of importance, no strong opinion on the order though